db/schema.rb

```diff
@@ -10,12 +10,12 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.2].define(version: 2025_02_03_164133) do
+ActiveRecord::Schema[7.2].define(version: 2025_02_17_105318) do
```

Comment:
Are all the changes in this file from this migration? It looks like some extra stuff has slipped in